types/time.go

```diff
@@ -1261,6 +1276,8 @@ func ParseDuration(sc *stmtctx.StatementContext, str string, fsp int8) (Duration
 				_, err = fmt.Sscanf(integeralPart, "%2d", &second)
 			case 1: // 0S
 				_, err = fmt.Sscanf(integeralPart, "%1d", &second)
+			case 0:
+				return ZeroDuration, nil
```

Comment:
`time(" ")` returns null in MySQL while this line let TiDB return the zero duration.